src/components/AddBook.js

```diff
@@ -64,6 +65,11 @@ const AddBook = () => {
             <option value="Economy">Economy</option>
             <option value="Action">Action</option>
             <option value="Science Fiction">Science Fiction</option>
+            <option value="Classics">Classics</option>
+            <option value="Comedy Fiction">Comedy Fiction</option>
+            <option value="Detective and Mystery">Detective and Mystery</option>
+            <option value="Fantasy">Fantasy</option>
+            <option value="Historical Fiction">Historical Fiction</option>
           </select>
           <button className={classes.button} type="submit">
```

Comment:
- [ ] Please update the `back-ground-color: #3e98c7`  and the `border: none` to match the original design